src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/CPP/DWriteWrapper/TextAnalyzer.cpp

```diff
@@ -20,7 +20,7 @@ namespace MS { namespace Internal { namespace Text { namespace TextInterface
         __in_ecount(length) const WCHAR* text,
         UINT32                           length,
         CultureInfo^                     culture,
-        Factory^                         factory,
+        IDWriteFactory*                  pDWriteFactory,
```

Comment:
I changed `Factory` to `IDWriteFactory` to be able to remove Factory from DirectWriteForwarder. `Factory` was only used to call `DWriteFactoryAddRef` and then use the returned `IDWriteFactory`. I moved the responsibility of calling `DWriteFactoryAddRef` to the caller the method.